e2e-tests/cypress/e2e/donor/donor-journey.cy.ts

```diff
@@ -277,4 +277,31 @@ describe("Identify a Donor", () => {
     cy.contains("National insurance number");
   });
 
+  it("lets you choose the post-office route", () => {
+    cy.visit("/start?personType=donor&lpas[]=M-XYXY-YAGA-35F0");
+
+    cy.get("input#POST_OFFICE").click();
+    cy.get(".govuk-button").contains("Continue").click();
+
+    cy.get("[id=PASSPORT]").click();
```

Comment:
This isn't essential, but you should be able to select these by content rather than attributes:
```suggestion
    cy.contains("Passport").click();
```
And you can chain in a CSS selector if the text appears multiple times on the page:
```suggestion
    cy.get("label").contains("Passport").click();
```